l10n_it_ricevute_bancarie/models/account/bank_statement.py

```diff
@@ -0,0 +1,53 @@
+# -*- coding: utf-8 -*-
+# Copyright 2018 Lorenzo Battistini - Agile Business Group
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+from odoo import models, api
+from odoo.osv import expression
+
+
+class AccountBankStatementLine(models.Model):
+    _inherit = 'account.bank.statement.line'
+
+    def get_move_lines_for_reconciliation(
```

Comment:
Here (https://github.com/odoo/odoo/blob/9032617120138848c63b3cfa5d1913c5e5ad76db/addons/account/models/account_bank_statement.py#L546) it is `api.multi`, should keep the decorator

Reply:
@SimoRubi you looked at the wrong method: https://github.com/odoo/odoo/blob/9032617120138848c63b3cfa5d1913c5e5ad76db/addons/account/models/account_bank_statement.py#L557